test/examples.js

```diff
@@ -0,0 +1,487 @@
+'use strict';
+
+const metasync = require('..');
+const fs = require('fs');
+const metatests = require('metatests');
+const ASYNC_TIMEOUT = 200;
+
+// Data Collector
+
+metatests.test('dataCollector / simple', test => {
+
+  const dataCollector = metasync
+    .collect(4)
+    .done((err, data) => {
+      test.error(err);
+      test.strictSame(Object.keys(data).length, 4);
+      test.end();
+    });
+
+  dataCollector.collect('user', null, { name: 'Marcus Aurelius' });
+
+  fs.readFile('README.md', test.mustCall((err, data) => {
+    dataCollector.collect('readme', err, data);
+  }));
+
+  fs.readFile('AUTHORS', test.mustCall((err, data) => {
+    dataCollector.collect('authors', err, data);
+  }));
+
+  setTimeout(test.mustCall(() => {
```

Comment:
I don't think you need to use `mustCall` for node/v8 functions(e.g. setTimeout, fs.readFile).